packages/manager/modules/carrier-sip/src/endpoints/translations/Messages_fr_FR.json

```diff
@@ -1,5 +1,6 @@
 {
-  "carrier_sip_endpoints": "SIP Endpoints",
+  "carrier_sip_endpoints_list_IP": "Liste d'IP autorisées pour les appels sortants",
```

Comment:
Sometimes we write `IPs` sometimes not...
Cf: https://github.com/ovh/manager/blob/7d34718274f5eae1335bd0b8f91af8745cb094d8/packages/manager/modules/carrier-sip/src/dashboard/translations/Messages_fr_FR.json#L11

Reply:
Changed in linked file :)

Reply:
**nitpick:** It is probably a personal preference but I find `IP address(es)` way more explicit.